src/common/date-provider.ts

```diff
@@ -17,13 +16,15 @@ export class DateProvider {
     }
 
     public static getUTCStringFromDate(date: Date): string {
-        return utc(date.toISOString()).format('YYYY-MM-DD h:mm A z');
+        const utcDateTime = DateTime.fromJSDate(date, { zone: 'utc' });
+        return utcDateTime.toFormat('YYYY-MM-DD h:mm A z');
```

Comment:
We recently had someone observe that UTC times are typically specified in 24-hour format; do we want to address that here?

Reply:
I wanted to try to minimize the number of format changes that weren't strictly required for the library conversion. Personally, I agree with [Luxon's writeup](https://moment.github.io/luxon/docs/manual/formatting.html#consider-alternatives) that we should either be using ISO8601 or a system-default locale string and not using our own, but I wanted to separate the "moment to luxon conversion" PR from any "format change PRs" to the extent possible